cpp/src/toppra/constraint.cpp

```diff
@@ -1,5 +1,7 @@
 #include <toppra/constraint.hpp>
 
+#include <toppra/geometric_path.hpp>
+
 namespace toppra {
 
 std::ostream& LinearConstraint::print(std::ostream& os) const
```

Comment:
How do you use this method? Look really neat. Would be great if you can provide a usage example!

Reply:
`std::cout << aLinearConstraintInstance << std::endl`. I have it on local changes but did not push it.

Reply:
This is very common in C++. See https://www.learncpp.com/cpp-tutorial/12-10-printing-inherited-classes-using-operator/ for some insights.

Reply:
That's neat! Thanks.